pyproject.toml

```diff
@@ -18,7 +18,7 @@ exclude = [
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
 numpy = "^1.24.1"
-pandas = "^1.5.3"
+pandas = ">=2.0,<3.0"
```

Comment:
You decided against >=2.0.0?

Reply:
Adding the "<3.0" can't possibly do any harm, and will avoid any nasty poetry surprises, should we pull in any dependencies in the future that happen to specify things this way.